src/VirtoCommerce.DemoSolutionFeaturesModule.Core/Models/Order/DemoOrderConfiguredGroup.cs

```diff
@@ -5,10 +5,11 @@
 
 namespace VirtoCommerce.DemoSolutionFeaturesModule.Core.Models
 {
-    public class DemoOrderConfiguredGroup: AuditableEntity, ICloneable
+    public class DemoOrderConfiguredGroup : AuditableEntity, ICloneable
     {
         public string ProductId { get; set; }
-        public ICollection<string> ItemIds { get; set; } = new List<string>();
+        public ICollection<string> ItemIds { get; set; }
+        public ICollection<DemoOrderLineItem> Items { get; set; }
```

Comment:
Why this collection isn't initialized?

Reply:
Because it is a common approach for presentation models in the platform.